x-pack/plugins/fleet/public/components/new_enrollment_key_modal.tsx

```diff
@@ -86,15 +94,18 @@ export const NewEnrollmentTokenModal: React.FunctionComponent<Props> = ({
       <form onSubmit={form.onSubmit}>
         <EuiFormRow
           label={i18n.translate('xpack.fleet.newEnrollmentKey.nameLabel', {
-            defaultMessage: 'Name',
+            defaultMessage: 'Token name',
           })}
         >
-          <EuiFieldText
-            required={true}
-            name="name"
-            autoComplete="off"
-            {...form.apiKeyNameInput.props}
-          />
+          <EuiFlexItem>
+            <EuiFieldText
+              name="name"
+              autoComplete="off"
+              placeholder="Enter a token name"
+              {...form.apiKeyNameInput.props}
+            />
+            <EuiFormHelpText>Token id will be used when this is left empty.</EuiFormHelpText>
```

Comment:
We probably want to translate that you can use the `<FormattedMessage>` component for that.

Reply:
Also I think we can define `helpText=` on the Form row component too https://elastic.github.io/eui/#/forms/form-layouts#form-and-form-rows

Reply:
thanks, will update